apps/src/sites/studio/pages/datasets/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DatasetList from '@cdo/apps/storage/levelbuilder/DatasetList';
+
+$(document).ready(function() {
```

Comment:
I was curious why we needed the document.ready here, so I googled it and couldn't find a clear answer besides React.render needs it. I saw some indication too that ReactDOM.render doesn't need the document.ready as it implicitly waits. Do you know if that's the case here?

Reply:
I'm not exactly sure, but I just followed the model of every other page in this directory, which all have `$(document).ready`